HelpAssistHer/help-assist-her
# General info

## Description
- Community Health Center Form mvp: on submit, general information value of CHC form will be logged in the browser console.
- it is accomplished by adding one new exportable variable (chcInputResource) and two components (form and register field).
- chcInputResource is an array of objects and arrays which will have all the information about fields need to show on the form with verification. 
- RegisterField component will accept props from its parent and will return a form element which is one verified field and one input field. 
 - form component will accept two variables, chcInputResource, and onSubmit.
- the form will loop through chcInputResource prop and dynamically generate RegisterField component which in result return each input element and its verification input element.
- onSubmit will only log the form value for now.
- it will not reset the form.
## Test Plan
 -  run yarn webpack
- go to http://localhost:4000/verification/
- click on Community Health Center
- Will see the form with general info and checkbox
- Fill the form 
- check the box 
- hit save progress
- see the console for form values 
- form will not reset
- DONE
